src/app/modules/shared_models/message_item_qobject.nim

```diff
@@ -111,15 +111,15 @@ QtObject:
   #   read = reactionIds
 
   proc pinned*(self: MessageItem): bool {.slot.} = result = ?.self.messageItem.pinned
-  QtProperty[int] bool:
+  QtProperty[bool] bool:
```

Comment:
@borismelnik but man, why did you leave `bool` as name here?

In my previous comment I said you should update to `QtProperty[bool] pinned:` instead `QtProperty[int] bool:`, also you should add name which is used on the qml side for other 2 cases.

Reply:
oh :D